packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -34,8 +34,8 @@ export function isCLIInstalled(): boolean {
 export function showCLINotInstalledMessage() {
   const showMessage = nls.localize(
     'sfdx_cli_not_found',
```

Comment:
```suggestion
    'salesforce_cli_not_found',
```
I think we should update this too